src/controllers/expensesController.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+function getAll(req, res) {
+  const {
+    userId,
+    category,
+    from,
+    to,
+  } = req.query;
+
+  const filteredExpenses = expensesService.getAll(userId, category, from, to);
+
+  res.statusCode = 200;
+  res.send(filteredExpenses);
+};
+
+function getOne(req, res) {
+  const { expenseId } = req.params;
+
+  const foundExpense = expensesService.getbyId(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundExpense);
+};
+
+function add(req, res) {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const foundUser = usersService.getbyId(userId);
+
+  if (!foundUser) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expensesService.create(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  );
```

Comment:
you can just pass the whole body here
```suggestion
  const newExpense = expensesService.create(req.body);
```